tests/builder/test_base.py

```diff
@@ -125,3 +159,88 @@ def count_markers(path, marker='# Edited by rpmlb'):
         edit_file(empty_spec_path, indicator)
         assert empty_spec_path.exists()
         assert count_markers(empty_spec_path) == 1
+
+
+def test_macros_are_added_correctly(macro_spec_path, prepared_macros):
```

Comment:
@junaruga 

>  Please change your function names "test_*" referring below lines.

I renamed the tests a bit more verbosely. Let me know if you would like more adjustments.